recipes-kernel/linux/linux-yocto-onl-linux-6.1.y/bisdn-kmeta/bsp/x86_64-intel/x86_64-intel.cfg

```diff
@@ -3521,6 +3527,7 @@ CONFIG_DMA_VIRTUAL_CHANNELS=y
 CONFIG_DMA_ACPI=y
 # CONFIG_ALTERA_MSGDMA is not set
 # CONFIG_INTEL_IDMA64 is not set
+CONFIG_INTEL_IOATDMA=m
```

Comment:
There's `features/dca/dca.scc` that sets this to =y, as well as `OCNFIG_DCA` and `CONFIG_IGB_DCA`. Not sure if this is totally releated.

Reply:
Works for me. Done.